pz_be_services/routers/v1/user_router.py

```
@@ -194,6 +193,110 @@ async def github_callback( code: str = None, db: Session = Depends(get_db)):
+            user_info = user.username
+            redirect_url = f"{FRONTEND_REDIRECT_URL}?access_token={access_token}&username={user_info}"
+            
+            
+            return RedirectResponse(redirect_url)
+            
+        except ValueError as e:
+            logger.error(f"Username generation failed: {str(e)}")
+            raise HTTPException(
+                status_code=400, 
+                detail="Unable to create user account with provided Google data"
+            )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)